Commands/Public/wiki.js

```diff
@@ -19,7 +19,10 @@ module.exports = async ({ Constants: { Colors } }, documents, msg, commandData)
 				embed: {
 					color: Colors.SOFT_ERR,
 					title: "What was that again? 📚🤓",
-					description: "The almighty Wikipedia could not find a result for your search query. Maybe try something different?",
+					description: "A result could not be found for your search query.",
```

Comment:
I prefer the old almighty text tbh
Sorry was luv u

Reply:
:blobheart: